src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1268,14 +1268,16 @@ static MethodSymbol getTopLevelMethod(MethodSymbol method)
             }
         }
 
+#nullable enable
         protected override int GetOrCreateSlot(Symbol symbol, int containingSlot = 0, bool forceSlotEvenIfEmpty = false)
```

Comment:
If it's possible for `symbol` to be null then the type should change to `Symbol?`.

Reply:
Shouldn't be possible, added an Assert to make sure in Commit 8.

---
In reply to: [447978511](https://github.com/dotnet/roslyn/pull/45563#discussion_r447978511) [](ancestors = 447978511)